charts/incubator/alfresco-dbp/minimal-values.yaml

```diff
@@ -0,0 +1,257 @@
+# Default values for Alfresco Process Services.
```

Comment:
Shouldn't specifying an alternate values file result in the contents being merged with the default values such that we don't have to repeat everything in this minimal file?

(@svidrascu, you may have mentioned something about a problem with this not working as expected?)

Reply:
yes, indeed we don't need everything, just the ones that collide and/or are new in minimal-values.yaml . Minimal values is now minimal indeed :)